src/components/copy-text/CopyText.tsx

```diff
@@ -48,7 +56,15 @@ export const CopyText = ({ value, copyTextHandler, copiedText = 'Copied!', ...re
       </ParagraphSmall>
       <StyledCopyIcon onClick={() => copyTextHandler(value)}>
         <StatefulTooltip triggerType="click" content={() => <Block>{copiedText}</Block>}>
-          <TertiaryButton>
+          <TertiaryButton
+            overrides={{
+              Root: {
+                style: {
+                  padding: '0',
```

Comment:
```suggestion
                  ...padding(),
```

to avoid issues with mixing long-hand and short-hand properties